router/src/lib.rs

```diff
@@ -781,7 +781,7 @@ pub(crate) struct Message {
     #[schema(example = "\"David\"")]
     pub name: Option<String>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub tool_calls: Option<ToolCall>,
+    pub tool_calls: Option<Vec<ChatCompletionMessageToolCall>>,
```

Comment:
I kind of prefer `ToolCall` in readability (chatCompletionMessage make it look like java imho).